pkgs/applications/science/logic/isabelle/components/isabelle-linter.nix

```diff
@@ -2,16 +2,26 @@
 
 stdenv.mkDerivation rec {
   pname = "isabelle-linter";
-  version = "Isabelle2021-1-v1.0.0";
+  version = "Isabelle2022";
 
   src = fetchFromGitHub {
     owner = "isabelle-prover";
     repo = "isabelle-linter";
-    rev = version;
-    sha256 = "0v6scc2rhj6bjv530gzz6i57czzcgpkw7a9iqnfdnm5gvs5qjk7a";
+    rev = "0424fc05426d5f7a23adf19ad08c690c17184e86";
+    sha256 = "02afbgmi195ibichjkpni2wjgjkszv7i6qkmmprwrmb4jd2wdvd5";
   };
 
-  installPhase = import ./mkBuild.nix { inherit isabelle; path = "${pname}-${version}"; };
+  buildPhase = ''
+    export HOME=$TMP
+    ${isabelle}/bin/isabelle components -u $(pwd)
+    ${isabelle}/bin/isabelle scala_build
+  '';
+
+  installPhase = ''
+    dir=$out/Isabelle${isabelle.version}/contrib/${pname}-${version}
+    mkdir -p $dir
+    cp -r * $dir/
```

Comment:
Do we really need to use this PATH and cannot use more FHS like ones?

Reply:
Yes, otherwise the Symlinkjoin for the plugins won't put them in the right spot